### ISSUE TITLE
Accessibility: pf wizard buttons are inaccessible via keyboard navigation

### ISSUE THREAD

**Issue by benjaminapetersen** [patternfly/angular-patternfly#603]
via [issue 406](https://github.com/openshift/origin-web-catalog/issues/406) in web console.

You cannot tab through the wizard content & then access the `cancel`, `next` buttons.

![2017-08-29 16 31 15](https://user-images.githubusercontent.com/280512/29842917-437e6f80-8cd8-11e7-8147-7db7055f8e91.gif)

- [ ] button is not actually a button, it is just a node styled like a button w/o a `tabindex`.
- [ ] you can also tab through content below the modal (on the page) while the modal is open.

**Comment by jeff-phillips-18** [comment]
Thanks @benjaminapetersen , would you like to contribute the solution?

**Comment by benjaminapetersen** [comment]
If I take a stab at it, it looks like each of the components in `src/wizard/wizard-buttons.js` need a template with a real `button` element.  That is prob not too hard.

To fix the tab issue with content under the modal, I would tend toward keeping track of the 1st and last "tabbable" item in the modal, add a `keyup` or `keydown` handler to the last & set focus on the first if `e.which === 9 && !e.shiftKey`.  What could get a  little ugly there is querying child view for the steps & updating the first element reference.  Using `require` may be cleaner...

Thoughts?  Reasonable direction?
